Cert updates

dockerymick opened the pull request:
https://github.com/open-cluster-management/backlog/issues/8446#issuecomment-761190425

dockerymick committed:
moved the obs cert section to the cert page

dockerymick committed:
left a comment for myself

dockerymick committed:
Update certificates.adoc

made more updates and left a comment

dockerymick committed:
Update certificates.adoc

gparvin commented on security/certificates.adoc:
```diff
@@ -32,24 +32,167 @@ oc get certificates.certmanager.k8s.io -n open-cluster-management
+
+CA certificates can be associated with Git channel that are a part of the {product-title-short} application management. See link:../manage_applications/configuring_git_channel.adoc[Configuring application channel and subscription for a secure Git connection] for more details.
+
+Helm channels allow you to disable certificate validation. Helm channels must be configured in development environments because disabling certificate validation introduces security risks.
```
The second sentence about helm channels is unclear.  Not all helm channels must be configured in development  environments -- only the ones where they chose to disable certificate validation.  Disabling certificate validation is an extra configuration option they have to chose -- only when they do that is the connection "risky".

dockerymick replied:
@gparvin Thanks for clarifying, I have updated the sentence. May you re-review the PR when you have an opportunity please?

gparvin reviewed (commented):
I added one comment about helm channels just to clarify it's the disabling of the certificate validation that makes it risky.

dockerymick committed:
Update certificates.adoc

Removed comment and clarified the helm channels sentence

gparvin reviewed (approved):
Thanks!

dockerymick commented on security/certificates.adoc:
```diff
@@ -32,24 +32,164 @@ oc get certificates.certmanager.k8s.io -n open-cluster-management
+[#channel-certificates]
+=== Channel certificates
+
+CA certificates can be associated with Git channel that are a part of the {product-title-short} application management. See link:../manage_applications/configuring_git_channel.adoc[Configuring application channel and subscription for a secure Git connection] for more details.
```
@rokej May you review this section since we are referencing application. We would like to point the user specifically to [Updating certificates and SSH keys section](https://github.com/open-cluster-management/rhacm-docs/blob/2.2_stage/manage_applications/configuring_git_channel.adoc#updating-certificates-and-ssh-keys). I think it would be beneficial to the user if we pointed them to the [Secret sample](https://github.com/open-cluster-management/rhacm-docs/blob/2.2_stage/manage_applications/secret_sample.adoc). The link would be at the end of  Updating certificates and SSH keys section. 


Do you have any thoughts on my proposed change?

rokej replied:
@dockerymick Yes, I agree with you on pointing the user specifically to Updating certificates and SSH keys section. However, the Secret sample does not contain samples about CA certificates or SSH keys. These samples are contained in other sections of ../manage_applications/configuring_git_channel.adoc. 

So probably we can just add a sentence like "see above sections for reconfiguring certificates and SSH keys" in Updating certificates and SSH keys section.

dockerymick replied:
@gparvin Based on @rokej suggestion, what do you think about pointing the user to the section, [Using custom CA certificates for a secure HTTPS connection](https://github.com/open-cluster-management/rhacm-docs/blob/8446/manage_applications/configuring_git_channel.adoc#using-custom-CA-certificates-for-secure-HTTPS-connection) instead of directing them to the Updating certificates and SSH keys section?

gparvin replied:
Sounds great!  Thanks!

dockerymick reviewed (commented):
@rokej may your review the Channel section of the security/certificates.adoc file please?

dockerymick commented:
> Thanks!

Thank you ! :) @gparvin

dockerymick committed:
Update certificates.adoc

dockerymick committed:
Update configuring_git_channel.adoc

Adding reference based on this [suggestion](https://github.com/open-cluster-management/rhacm-docs/pull/1686#discussion_r567873393)

dockerymick committed:
Update certificates.adoc

dockerymick committed:
Ran spellcheck and updated link reference
